time-manager-ui/src/app/structure/notices/notice/notice.component.ts

```diff
@@ -13,6 +13,7 @@ export class NoticeComponent implements OnInit {
   @Input() date!: string;
   @Output() deletedTask = new EventEmitter<any>();
   @Output() updatedTask = new EventEmitter<any>();
+  @Output() duplicatedTask = new EventEmitter<any>();
```

Comment:
I suppose it should be better to use `new EventEmitter<Task>()` instead of `new EventEmitter<any>(); `